tests/TestGUI.py

```diff
@@ -107,7 +108,7 @@ def __show_data_and_confirm(test_data_path: str):
         # Setup the window and canvas
         popup = tk.Toplevel()
         popup.wm_title("Data Confirmation")
-        popup.state('zoomed')
+        popup.state('zoomed')  # TODO unix-based systems use -zoomed
```

Comment:
Please fix this so we don't have to keep changing every commit.

Reply:
Yep will fix

Reply:
Fixed in branch dev-bugFixes